index.html

```diff
@@ -119,264 +119,156 @@
       </p>
-    </section>
-
-    <!-- Use Cases -->
-    <section id="profile-use-cases">
-      <h2>Use Cases and Requirements</h2>
       <p>
-        A set of over 30 use cases for the Web of Things were contributed by stakeholders
-        from multiple industries for various application domains.
-        These have been published in the WoT Use Cases and Requirements document [[wot-usecases]].
+        The Thing Description specification is designed to be protocol-agnostic 
+        and flexible enough to describe a wide range of existing ("brownfield") 
+        IoT devices, rather than specifying a fixed protocol or application 
+        programming interface (API) which all devices must implement. The downside
+        of this open ended flexibility is that it makes ad-hoc interoperability on
+        the Web of Things very difficult, because it's nearly impossible to create
+        a single WoT <a href="#dfn-consumer">Consumer</a> [[wot-architecture]] 
+        which is guaranteed to be able to communicate with any
+        <a href="#dfn-thing">Web Thing</a> [[wot-architecture]] out of the box.
       </p>
       <p>
-        Based on these use cases a set of requirements have been derived which drive the development
-        of the W3C Web of Things specification family.
-        Several of these domains require easy integration of devices from multiple vendors,
-        in other words, out-of-the-box interoperability.
-        However, the descriptive approach taken by the WoT specifications generally leads
-        to a large variety of different protocols and data formats,
-        which can work against out-of-the box interoperability.
+        This specification is designed to complement the Thing Description 
+        [[wot-thing-description]] specification, by defining a 
+        <a href="#profiling-mechanism">Profiling Mechanism</a> and a set of 
+        <a href="#dfn-profile">Profiles</a> which enable 
+        <a href="#out-of-the-box-interoperability">out-of-the-box
+        interoperability</a> between <a href="#dfn-thing">Web Things</a> and their
+        <a href="#dfn-consumer">Consumers</a>.
       </p>
       <p>
-        For example, a WoT Thing Description (TD) can in theory include a description based on a
-        networking protocol unknown to a device that wishes to connect to it.
-        To ensure interoperability without additional customization (e.g. by writing software or
-        performing complex setup or configuration steps),
-        the range of such choices needs to be limited to a finite set so that a consumer
-        of a Thing Description can be sure it will be able to interact with any possible Thing.
-        A finite set of customization choices is also important for implementing devices with a fixed code base.
-        Defining such constraints leads to the profile mechanism and the HTTP Basic Profile.
+        Profiles are designed specifically for new ("greenfield") implementations 
```

Comment:
This is a bit too narrow, since situations where both things and consumers are implemented from the ground up, will rarely happen.

There are use cases and business relevant scenarios, where it is perfectly fine to define a profile that interacts with existing consumers, see also the discussion about cloud events.

Reply:
> This is a bit too narrow, since situations where both things and consumers are implemented from the ground up, will rarely happen.

This is the only scenario in which profiles make sense. It is extremely unlikely that an existing Consumer would already be compatible with any of the profiles we have defined so far.

> There are use cases and business relevant scenarios, where it is perfectly fine to define a profile that interacts with existing consumers, see also the discussion about cloud events.

Having studied the CloudEvents specifications, even if we designed the Webhooks profile to follow those specifications it still wouldn't guarantee out-of-the-box interoperability with existing CloudEvents consumers. Those specifications are not prescriptive enough (e.g. they allow any HTTP verb to be used to subscribe to an event) and they leave a lot of details unspecified (e.g. the subscription payload format, or mutual authentication).

The only way an existing Consumer implementation could be conformant with a profile would be if we specifically designed the profile around one existing implementation (e.g. an Oracle cloud service which consumes CloudEvents). That is not the point of profiles, and would only benefit Oracle.

If we constrain ourselves to making profiles compatible with existing systems then we will have a sub-optimal design for no tangible benefit.

I stand by the statement that profiles are designed for greenfield implementations since that's the key factor which differentiates them from protocol binding templates.